src/detail-page/BackButton.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from 'styled-components';
+import Typography from '@material-ui/core/Typography';
+import ArrowBackIosIcon from '@material-ui/icons/ArrowBackIos';
+import { Button } from '@material-ui/core';
+import { useHistory } from 'react-router-dom';
+
+const BackButtonStyle = styled(Button)`
+margin: ${props => props.theme.spacing(2)}px 0px;  
```

Comment:
looks good now :) the only thing - we don't need to add `px` when value is 0

Reply:
done